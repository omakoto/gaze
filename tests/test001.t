#!/bin/bash

set -e
cd "$(dirname "$0")"

. ./common.bash

../bin/gaze --repeat 1 --width 40 --height 5 yes
