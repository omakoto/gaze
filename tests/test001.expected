[2J[?25l[1;1H           2017/07/13 19:40:00.000 -0700
[2;1Hy
[3;1Hy
[4;1Hy
[5;1Hy